app/Providers/AppServiceProvider.php

```diff
@@ -13,7 +13,10 @@ class AppServiceProvider extends ServiceProvider
      */
     public function register()
     {
-        //
+        if ($this->app->isLocal()) {
+            $this->app->register(\Laravel\Telescope\TelescopeServiceProvider::class);
```

Comment:
Please simplify FQN.
(you can press Alt+Enter on class name and select proper option)